runtime/runtime-core/jvm/test/aws/smithy/kotlin/runtime/content/ByteStreamJVMTest.kt

```diff
@@ -130,4 +131,18 @@ class ByteStreamJVMTest {
 
         assertEquals(1024, stream.contentLength)
     }
+
+    @Test
+    fun `can create byte stream from empty file using createTempFile`() = runTest {
+        val file = Files.createTempFile(null, null)
+        val byteStream = file.asByteStream()
+        assertEquals(0, byteStream.contentLength)
+    }
+
+    @Test
+    fun `can create byte stream from empty file using RandomTempFile`() = runTest {
+        val file = RandomTempFile(sizeInBytes = 0)
+        val byteStream = file.asByteStream()
+        assertEquals(0, byteStream.contentLength)
+    }
```

Comment:
**Comment**: These `File.asByteStream` tests look good but we're missing tests for `Path.asByteStream`.